core/types/Interfaces.ts

```diff
@@ -284,10 +284,10 @@ export interface MenuItem {
     className?: string;
 
     /** Executed when the user clicks the menu item. */
-    actionFn?: (e: MouseEvent | PointerEvent) => void;
+    actionFn?: (e: MouseEvent | PointerEvent, contextMenuEvent?: MouseEvent | PointerEvent) => void;
 
```

Comment:
So the first event `e` is the click on a specific item within the menu, and the new second event `contextMenuEvent` is the right-click that caused the menu to appear in the first place?